releasenotes/notes/remove-deprecated-optimizer-methods-d580a07112ccaa2d.yaml

```
@@ -0,0 +1,17 @@
+---
+upgrade:
+  - |
+    Remove the `optimize` method from the :class:`~qiskit.algorithms.optimizers.Optimizer`
+    classes, which is superseded by the :meth:`~.algorithms.optimizers.Optimizer.minimize` method as direct replacement.
+    The one exception is :class:`~qiskit.algorithms.optimizers.SPSA`, where the
+    deprecation warning was not triggered so the method there is still kept.
+deprecations:
+  - |
+    The :meth:`.SPSA.optimize` method is deprecated in
+    favor of :meth:`.SPSA.minimize`, which can be used
+    as direct replacement. Note that the `minimize` method returns a complete result
```

```suggestion
    as direct replacement. Note that this method returns a complete result
```
(or crossref it, or use two backticks ` ``minimize`` `)

---

done in [cf09910](https://github.com/Qiskit/qiskit-terra/pull/8123/commits/cf099109ac17d436446ec7efaf9787ebc94d7d2f)